index.html

```diff
@@ -3,12 +3,28 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Hello Front-End</title>
+  <title>Tic-Tac-Toe!</title>
   <link rel="stylesheet" type="text/css" href="css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans|Qwigley|Special+Elite" rel="stylesheet">
 </head>
 <body>
-  <h1>Hello Front-End</h1>
-
+  <h1>Tic Tac Toe</h1>
+  <p class="rules">Hopefully you already know how to play Tic Tac Toe but incase you don't here ya go: Grab a friend and take turns marking different squares on the board below. Player one will be X and player two  will be O. To win you must get three in a row! <em>Good luck!</em></p>
+  <p id="turn">Player one, your turn!</p>
+  <h1 class="winner" id="winner">We have a winner!</h1>
+  <div class="board" id="board">
+    <div class="row1 empty resetClass" id="a1">__</div>
+    <div class="row1 empty resetClass" id="b1">__</div>
+    <div class="row1 empty resetClass" id="c1">__</div>
+    <br>
```

Comment:
Instead of <br>, consider wrapping each array of div elements with a div to create rows. This will help you to target entire rows in the future if desired.